euvm/README.md

```diff
@@ -0,0 +1,43 @@
+## About eUVM
+
+eUVM is an opensource implementation of IEEE UVM-1800.2-2020 standard in the D Programming Language.
+
+## About the RISCV-DV eUVM port
+
+The RISCV-DV eUVM port is a line-by-line translation of the RTSCV-DV SystemVerilog implementation. Except for functional coverage (a work in progress), all other RISCV-DV features have been implemented in eUVM port.
```

Comment:
Typo: `RTSCV-DV` should be `RISCV-DV` (yeah, a bit pedantic).

Reply:
Thanks, Mike. Fixed now.